pkg/controller/controller_suite_test.go

```diff
@@ -51,20 +52,122 @@ var (
 	controllerKindMachine = v1alpha1.SchemeGroupVersion.WithKind("Machine")
 )
 
-func newMachineSetFromMachineDeployment(machineSetCount int, machineDeployment *v1alpha1.MachineDeployment, replicas int32, statusTemplate *v1alpha1.MachineSetStatus) *v1alpha1.MachineSet {
-	return newMachineSetsFromMachineDeployment(1, machineDeployment, replicas, statusTemplate)[0]
+func newMachineDeployment(
+	machineDeploymentCount int,
```

Comment:
Why do we need this `machineDeploymentCount` here?

Reply:
You are right, we don't need it. Have fixed it here - https://github.com/gardener/machine-controller-manager/pull/251/commits/d99cfa1f836c9e50d8396493c4e20487a66612f0